py/picca/fitter2/chi2.py

```diff
@@ -309,6 +309,13 @@ def export(self):
             g.attrs['chi2'] = d.chi2(self.k, self.pk_lin, self.pksb_lin, self.best_fit.values)
             fit = g.create_dataset("fit", d.da.shape, dtype = "f")
             fit[...] = d.best_fit_model
+            if d.bb != None:
+                gbb = g.create_group("broadband")
+                for k,bbs in d.bb.items():
+                    for bb in bbs:
+                        bband = gbb.create_dataset(bb.name, 
+                                d.da.shape, dtype = "f")
+                        bband[...] = bb(d.r, d.mu, **self.best_fit.values)
```

Comment:
@ngbusca, why do you need to use pointers here (`**`) when few lines above we don't.
Simply to understand,

Reply:
@londumas these are not pointers, what this does is it passes the items of the dictionary as keywords to the `bb` function. 

See https://stackoverflow.com/questions/36901/what-does-double-star-asterisk-and-star-asterisk-do-for-parameters